lib/workers/repository/update/pr/changelog/bitbucket/index.ts

```diff
@@ -69,6 +81,10 @@ export async function getReleaseNotesMd(
   return { changelogFile: changelogFile.path, changelogMd };
 }
 
+function getFilenameFromPath(filePath: string): string {
+  return filePath.split('/').pop()!;
```

Comment:
use `path.basename` and inline 

https://nodejs.org/api/path.html#pathbasenamepath-suffix

Reply:
Thank you - updated to use `path.basename`